src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -1,34 +1,80 @@
+/*
+This component contains temporary hard coding to fill data that should be pulling from the database.
+- currently those BE endpoints and DB data structures are not finished. 11-07-21
+Once this component pulls data from the database, change the parameters in the render section so that they pull from props and this page will be complete.  
+
+As BE endpoints and DB schema are finished:
+[] remove hardcode virtual player data, pull this data from BE and DB. Virtual player creation & data should be on BE, not FE
+[] remove hardcode user data, pull this data from props or global state
+[] remove hardcode point data, pull this data from props or global state
+[] remove hardcode story / drawing submission data, pull this data from BE and DB
+[] ensure that handleSubmitPoints passes the correct data to BE
+[] remove all comments when component is complete and ready for production
+
+Complete:
+[x] styling matches Figma 
+[x] counter boxes increment and decrement, saving value to component state
+[x] component level state saves and renders correctly ---> storyOnePoints, storyTwoPoints, illustrationOnePoints, illustrationTwoPoints
+*/
+
 import React, { useEffect, useState, useCallback } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useAuth0 } from '@auth0/auth0-react';
 import { Header } from '../../common';
-import { Row, Col, InputNumber, Button, notification } from 'antd';
-import { QuestionCircleOutlined } from '@ant-design/icons';
+import Footer from '../GamePlay/Footer';
+import { Button, notification } from 'antd';
+import {
+  ZoomInOutlined,
+  CaretUpOutlined,
+  CaretDownOutlined,
+} from '@ant-design/icons';
 import { connect } from 'react-redux';
 import { submitPoints } from '../../../api/index';
 
 import { SubmissionViewerModal } from '../../common';
-import { InstructionsModal } from '../../common';
-import { modalInstructions } from '../../../utils/helpers';
+
+// import hero images - TEMPORARY HARD CODE - remove when pulling state ( avatarID ) from backend
```

Comment:
Add temporary hard code data to allow component to render properly.  This will be removed once component pulls data from props or BE.